ui/ducks/bridge/selectors.ts

```diff
@@ -691,6 +693,13 @@ export const getValidationErrors = createDeepEqualSelector(
   },
 );
 
+export const getIsQuoteExpired = createSelector(
```

Comment:
Tried to use `createDeepEqualSelector` here, but the component was not updating, maybe did not recognised the quotesLastFetchedMs?